### ISSUE TITLE
Prettier not breaking the lines correctly

### ISSUE THREAD

**Issue by danprates** [sajansharmanz/prettier-plugin-edgejs#3]
Hi, thank you very much for doing this job.

About the issue:
I notice that some times the prettier was not breaking the line correctly
And I don't know if is there any configuration to fix this or if it's a improvement

![improvements](https://github.com/user-attachments/assets/6c91a769-3ba7-45db-85f5-ddc9e72ac30d)

My `prettier.config.js`:
```js
import adonisPrettierConfig from '@adonisjs/prettier-config' assert { type: 'json' }

export default {
  ...adonisPrettierConfig,
  plugins: ['prettier-edgejs'],
}
```

**Comment by sajansharmanz** [comment]
@danprates thanks for raising the bug! The <p> tag defo isn't behaving as intended here.

I can hopefully take a look at it this weekend and find a fix.

I'll keep you posted.

**Comment by sajansharmanz** [comment]
@danprates this issue should be resolved in the latest `prettier-edgejs@0.2.19` release
